I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 27, in script call
    call ch0_main from _call_ch0_main
  File "game/script-ch0.rpy", line 8, in script
    $ ChangeTime("Daytime")
  File "game/script-ch0.rpy", line 8, in <module>
    $ ChangeTime("Daytime")
  File "game/hud.rpy", line 36, in ChangeTime
    currentTime = newTime
NameError: name 'newTime' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 27, in script call
    call ch0_main from _call_ch0_main
  File "game/script-ch0.rpy", line 8, in script
    $ ChangeTime("Daytime")
  File "C:\Program Files (x86)\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Program Files (x86)\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script-ch0.rpy", line 8, in <module>
    $ ChangeTime("Daytime")
  File "game/hud.rpy", line 36, in ChangeTime
    currentTime = newTime
NameError: name 'newTime' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
Genshou Girls Festival 1.0
Thu Dec  8 02:19:37 2022
